packages/material/scss/slider/_variables.scss

```diff
@@ -37,10 +37,10 @@ $kendo-slider-button-focus-shadow: null !default;
 
 /// The thickness of the Slider track.
 /// @group slider
-$kendo-slider-track-thickness: 2px !default;
+$kendo-slider-track-thickness: 4px !default;
```

Comment:
We could also replace this with a value from the spacing module, as done with the other variables below